src/utilities/idf/ValidityEnums.hpp

```diff
@@ -39,20 +39,23 @@ namespace openstudio {
 /** \class StrictnessLevel
  *  \brief Specifies the desired amount of validity.
  *
- *  \li None - (Almost) anything goes.
+ *  \li None - Anything goes.
+ *  \li Minimal - (Almost) anything goes.
  *  \li Draft - Non-null data is valid.
  *  \li Final - Required and unique data has correct cardinality.
  *
  *  See the OPENSTUDIO_ENUM documentation in utilities/core/Enum.hpp. The actual macro call is:
  *  \code
 OPENSTUDIO_ENUM(StrictnessLevel,
   ((None))
+  ((Minimal))
   ((Draft))
   ((Final))
 );
  *  \endcode */
 OPENSTUDIO_ENUM(StrictnessLevel,
   ((None))
+  ((Minimal))
```

Comment:
I know I'm the one that proposed changing the current `None` to `Minimal`, but that's actually a breaking change... Since it's in the public api: a user is free to instantiate a workspace themselves eg: `w = OpenStudio::Workspace.new("None".to_StrictnessLevel, "EnergyPlus".to_IddFileType)`

I don't know how much we care though.

An alternative would be to add another name completely, like `Unsafe`, `FallThrough`, or something. Otherwise if we're good with this, we should make it part of the release notes (and label  this PR as `API Change`)